app/controllers/appointment_summaries_controller.rb

```diff
@@ -31,13 +33,11 @@ def preview
   def appointment_summary_params
     params
       .require(:appointment_summary)
-      .permit(:title, :first_name, :last_name,
-              :date_of_appointment,
-              :guider_name, :guider_organisation,
-              :address_line_1, :address_line_2, :address_line_3, :county, :town, :postcode, :country,
-              :has_defined_contribution_pension,
-              :supplementary_benefits, :supplementary_debt,
-              :supplementary_ill_health, :supplementary_defined_benefit_pensions, :format_preference,
-              :appointment_type)
+      .permit(*appointment_summary_editable_column_names)
+      .merge(user: current_user)
+  end
+
+  def appointment_summary_editable_column_names
```

Comment:
I have reverted this to the list of params which feel more dirty, but avoid knowledge of the AR object and keep inline with rails conventions.

As a side note, I agree Strong params don't make this any easier.